pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -79,7 +73,8 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 		//   https://github.com/opencontainers/runtime-spec/blob/ad53dcdc39f1f7f7472b10aa0a45648fe4865496/config-windows.md#cpu
 		//   If both CpuWeight and CpuMaximum are set - ContainerD catches this invalid case and returns an error instead.
 
-		cpuMaximum := 10000 * cpuLimit.MilliValue() / int64(runtime.NumCPU()) / 1000
+		numCPU := int64(winstats.ProcessorCount())
+		cpuMaximum := 10000 * cpuLimit.MilliValue() / numCPU / 1000
```

Comment:
What's with the multiplication by `10000`, then division by `1000`; can simplify to just multiplication by `10`?

Reply:
I'm not sure if I should do this in this PR since the PR's goal is just about the CPU count. However since I added unit tests while extracting the calculation, I also opted for adding your proposed simplification 👍

Reply:
I like the simplification 👍🏽 Do we need a comment about it in case we come back to this and the calculation is not immediately obvious to us?

Reply:
I'm not quite sure how I'd phrase that. Could you suggest something please? Thanks 🙂 

Whether it's needed or not is debatable I guess. git history is there and the relevant docs on microsofts side are there, too. But over time it'll become harder to find the original reason why it got changed. A comment might help in this case.